src/index.html

```diff
@@ -7,8 +7,85 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Moyo header</h1>
+  <body class="body">
+    <header class="header">
+      <a
+        class="logo" 
+        href="http://localhost:8080/"
+      >
+        <img src="/images/logo.png" alt="Logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__link-container">
+            <a 
+              class="nav__link is-active" 
+              href="http://localhost:8080/"
```

Comment:
Fix the href attribute everywhere, please

https://user-images.githubusercontent.com/99403198/212060518-2eaf5f16-bd20-4214-aff2-30b77d6e571c.mov

```suggestion
              href="#Apple"
```